maquetado/scss/_variables.scss

```diff
@@ -4,3 +4,6 @@ $font-family: 'Roboto', sans-serif;
 $text-decoration-height: 90px;
 $text-decoration-width: 70px;
 $text-big: 24px;
+$text-medium: 16px;
+$tech-logo-size: 100px;
+$tech-decoration-offset: 11px;
```

Comment:
Can `11px` be `10px`?

Reply:
That can, but for be accurate I should consider the `border-width` of `tech::before`.

Reply:
Ok, but in next opportunities prefer measurements in 5 multiples.